### PR TITLE
Generate VS nuget package for 2.1

### PR DESCRIPTION
**Pull request by jkotalik**
Fixes https://github.com/aspnet/AspNetCore/issues/11071. This will also be in 2.2 once we merge up.

### CODE REVIEW

**Comment by jkotalik** [comment]
Builds are here: http://aspnetci/viewLog.html?buildId=659155&buildTypeId=Releases_21Public_WindowsInstallers&tab=artifacts#!fk6iluux85. Only difference is that the Major and Minor versions are actually set 😄

File: src/Installers/Windows/SharedFramework/SharedFramework.wixproj
Comment (dougbu) [inline src/Installers/Windows/SharedFramework/SharedFramework.wixproj]: File a bug to track this if the TODO isn't resolved before you merge

**Review by dougbu — APPROVED** [review]
@anurse I suggest we at least let Shiproom know before merging this. I suspect we're done building 2.x for the next releases and that this will need to wait 'til August.

**Comment by dougbu** [comment]
Note: I'm approving because the fix looks fine but do not merge without additional authorization

**Comment by anurse** [comment]
Yes, the branches are ❌ Closed ❌ right now.

File: src/Installers/Windows/SharedFramework/SharedFrameworkPackage.nuspec
Comment (anurse) [inline src/Installers/Windows/SharedFramework/SharedFrameworkPackage.nuspec]:
```suggestion
        <id>VS.Redist.Common.AspNetCore.SharedFramework.$ARCH$.$MAJOR$.$MINOR$</id>
```

Leading space is *probably* not problematic but is weird.

File: src/Installers/Windows/SharedFramework/SharedFrameworkPackage.nuspec
Comment (anurse) [inline src/Installers/Windows/SharedFramework/SharedFrameworkPackage.nuspec]:
```suggestion
        <title>VS.Redist.Common.AspNetCore.SharedFramework.$ARCH$.$MAJOR$.$MINOR$</title>
```

**Review by analogrelay — APPROVED** [review]

**Comment by jkotalik** [comment]
@anurse feel free to merge this whenever 😄

**Pull request merged**
